jenkins/master/Dockerfile.ubi8

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-jenkins
```

Comment:
IIRC this has no actual effect due to https://github.com/opendevstack/ods-core/blob/master/jenkins/ocp-config/build/bc.yml#L119.

Reply:
True, but for GitHub actions test it does use it right?